src/v0.1.0/properties/structures/_cheminfo_smiles.yaml

```diff
@@ -0,0 +1,69 @@
+$$schema: "https://schemas.optimade.org/meta/v1.2/optimade/property_definition"
+$id: "https://schemas.optimade.org/namespaces/cheminformatics/v0.1/properties/structures/_cheminfo_smiles"
+title: "SMILES (Simplified Molecular Input Line Entry Specification) representation of the structure"
+x-optimade-type: "string"
+x-optimade-definition:
+  kind: "property"
+  version: "0.1.0"
+  format: "1.2"
+  name: "_cheminfo_smiles"
+  label: "_cheminfo_smiles_structures"
+type:
+  - "string"
+  - "null"
+description: |-
+  SMILES (Simplified Molecular Input Line Entry System) representation of the structure.
+  Values are RECOMMENDED to adhere to the OpenSMILES specification v1.0 (http://opensmiles.org/opensmiles.html).
+  When structures or their parts cannot be unambiguously represented in SMILES according to OpenSMILES recommendations, using the guidelines from Quirós et al. 2018 (https://doi.org/10.1186/s13321-018-0279-6) is RECOMMENDED.
+  Providers MAY canonicalize (i.e., use rules to establish the stable order of atoms) the produced SMILES representations, but this is not mandatory and no particular set of rules is recommended.
+  Generally, providers SHOULD NOT change the representation more frequently than the structure itself is modified.
+examples:
+  - "c1ccccc1"
+  - "CN1C=NC2=C1C(=O)N(C(=O)N2C)C"
+x-optimade-metadata-definition:
+  title: "Metadata for the _cheminfo_smiles field"
+  description: "This dictionary contains the per-entry metadata for the _cheminfo_smiles field."
+  x-optimade-type: "dictionary"
+  type:
+    - "object"
+    - "null"
+  properties:
+    _cheminfo_standard_name:
+      description: |-
+        A name of the standard the given SMILES string conforms to.
+        Takes value from an enumerator containing: 'IUPAC SMILES+' (https://github.com/IUPAC/IUPAC_SMILES_plus), 'OpenSMILES' (http://opensmiles.org/opensmiles.html) and 'other' (none of the above).
```

Comment:
We might also want to mention that the 'other' value should be accompanied by an entry in the '_cheminfo_standard_name_other' field.